src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -234,4 +234,12 @@ public void deleteExpiredEvent() {
             p.removeExpiredEvent();
         }
     }
+
+    /**
+     * Returns current filtered schedule of the address book.
+     */
```

Comment:
Another alternative is you can use {@inheritdoc}. But doesn't affect so its okay!